src/renderer/components/+helm-releases/releases.tsx

```diff
@@ -134,10 +134,12 @@ class NonInjectedHelmReleases extends Component<Dependencies & Props> {
         return releases.get().filter((release) => release.isSelected);
       },
 
-      removeSelectedItems() {
-        return Promise.all(
-          releases.get().filter((release) => release.isSelected).map((release) => release.delete()),
-        );
+      pickOnlySelected: (releases) => {
+        return releases.filter(release => release.isSelected);
+      },
+
+      removeItems: async (releases) => {
+        await Promise.all(releases.map(release => release.delete()));
```

Comment:
Port forwards may need a similar fix? I just removed all of mine thinking I was removing just one

Reply:
Ah thanks for the catch